userspace/falco/falco_metrics.cpp

```diff
@@ -174,7 +171,7 @@ std::string falco_metrics::to_text(const falco::app::state& state)
 		{
 			const stats_manager& rule_stats_manager = state.engine->get_rule_stats_manager();
 			const indexed_vector<falco_rule>& rules = state.engine->get_rules();
-			auto metric = libs_metrics_collector.new_metric("rules.matches_total",
+			auto metric = libs_metrics_collector.new_metric("rules_matches_total",
```

Comment:
Item 3 https://github.com/falcosecurity/falco/issues/3271#issuecomment-2206918760 (removing raw_name would be for the next release for sure alongside me moving the metric name sanitization to libs and I will also add a labels name sanitization method to libs).

Reply:
CC @Issif 

@falcosecurity/falco-maintainers do we want a patch release for these minor fixes? This would allow us to create more powerful Grafana dashboards and accelerate the deprecation for the falco-exporter.

Reply:
I think a patch release with this + the centos7 EOL fix would be fine.
The only problem is the time constraint since we are nearing august and most maintainers will be off during that month!

Reply:
ACK, also re-pushed to actually not remove the sha256 :woman_facepalming: